FogCreek/Glitch-Community
# Buttercup nickel: Update collections to show latest projects + fix aria-labels on my stuff button

## Links
* Remix link: https://buttercup-nickel.glitch.me/

## GIF/Screenshots:
- wrap bookmark button in a hidden checkbox and fix a11y labels so it's more clear what's happening on a screen reader

![](http://g.recordit.co/usCaLYNDhx.gif)

- keeping state a little bit more up to date by reloading what projects are in a collection after clicking the my stuff button: http://g.recordit.co/W7p6bLxOv3.gif (can't get this to show inline for some silly reason)

## Changes:
* reload collections after we update them
* pass in project names and put bookmark button into a hidden checkbox rather than just using aria-pressed on a button to make the state more clear. 

## How To Test:
* enable my stuff
* try adding/removing a project from my stuff, see if state seems up to date
* `command+ f5` to turn on voiceover, use `capslock+ u` and `arrow left/right` to get ensure you can perceive all the "my stuff" buttons even if they are not there. 

## Feedback I'm looking for:
- better ways to do it? bugs I'm not seeing? I still find stale state throughout the site from time to time...but this seems like an improvement. 

## Things left to do before deploying:
- [x] running this tooltips stuff by cassey
- [x] fix merge conflicts
